content/en/language_clients/python/github_action.md

```diff
@@ -0,0 +1,24 @@
+---
+type: docs
+category: Python
+menuTitle: GitHub Action
+title: GitHub Action
+weight: 50
+---
+
+## GitHub Actions
+
+`sigstore-python` has [an official GitHub Action](https://github.com/sigstore/gh-action-sigstore-python)!
+
+You can install it from the [GitHub Marketplace](https://github.com/marketplace/actions/gh-action-sigstore-python), or add it to your CI manually:
+
+```yaml
+jobs:
+  sigstore-python:
+    steps:
+      - uses: sigstore/gh-action-sigstore-python@v0.2.0
```

Comment:
This is referencing a really old version of the action -- is there a reason for that?

If not, I'd suggest the latest tag instead:

```suggestion
      - uses: sigstore/gh-action-sigstore-python@v3.0.0
```

Reply:
Oh, I see the ancient version is also in sigstore-python's README 🤦 -- I'll fix that.